lib/netext/grpcext/conn.go

```diff
@@ -174,7 +174,7 @@ func (c *Conn) Close() error {
 }
 
 type statsHandler struct {
-	vu modules.VU
+	getState func() *lib.State
```

Comment:
nit: Do we need this to be a function :thinking: 

I guess for future proofing in case we start chaning what lib.State is returned. But IIRC we basically return the same struct eitherway.

We do need it to be a function on the VU as it is an interface. But I wonder if we can skip the inderection here :thinking:

Reply:
I'd slightly prefer to keep the function, just so we keep the same sort of API structure and in case it might matter in the future, but I can make the change if others feel the same as you.

FWIW, I even considered adding another interface with just a `State() *lib.State` function in `grpext/` and continuing to pass the `VU` directly, or even moving the whole `DefaultOptions` function back in the `js/` package. The current approach seemed like a nice middle ground :sweat_smile: